btcstaking-tracker/stakingeventwatcher/expected_babylon_client.go

```diff
@@ -111,13 +112,20 @@ func (bca *BabylonClientAdapter) IsDelegationVerified(stakingTxHash chainhash.Ha
 func (bca *BabylonClientAdapter) ReportUnbonding(
 	ctx context.Context,
 	stakingTxHash chainhash.Hash,
-	stakerUnbondingSig *schnorr.Signature) error {
+	stakeSpendingTx *wire.MsgTx,
+	inclusionProof *btcstakingtypes.InclusionProof) error {
 	signer := bca.babylonClient.MustGetAddr()
 
+	var stakeSpendingTxBuf bytes.Buffer
+	if err := stakeSpendingTx.Serialize(&stakeSpendingTxBuf); err != nil {
```

Comment:
on `bbn` (`bbn "github.com/babylonlabs-io/babylon/types"`)we have this helper - `bbn.SerializeBTCTx(`